manifests/install.pp

```diff
@@ -1,5 +1,5 @@
-# Class: squid::install
-
+# @summary
+#   Installs the squid package
 class squid::install  {
```

Comment:
Could you add `@api private` as well? Same goes for `params.pp`, `service.pp` and other classes that aren't supposed to be used directly.